devtools/conda-envs/test_env.yaml

```diff
@@ -6,46 +6,47 @@ channels:
   - omnia
   - openeye
 dependencies:
-  - python >=3.6
-  - pip
-  - setuptools
+  - arsenic
+  - autograd
+  - click
+  - cloudpathlib-s3
+  - coverage
+  - dask
+  - dask-jobqueue
+  - dicttoxml
+  - dicttoxml
+  - distributed
   - fire
   - joblib
-  - numpy >=1.14
-  - scipy
-  - numexpr
-  - autograd
-  - pymbar <4
-  - openmm >=7.6
-  - openmoltools # may need to sort out ambermini/ambertools/parmed dependencies (we don't want ambertools)
-  - openmmtools >=0.21.2 # may need to sort out ambermini/ambertools/parmed dependencies
-  - numba >=0.55.2
-  - netcdf4
+  - lxml
   - matplotlib
-  - seaborn
   - mdtraj
-  - pdbfixer
-  - lxml
+  - netcdf4
   - networkx >=2.0
+  - numba >=0.55.2
+  - numexpr
+  - numpy >=1.14
   - openeye-toolkits
-  - dask
-  - dask-jobqueue
-  - distributed
-  - progressbar2
-  - parmed # may need to sort out ambermini/ambertools/parmed dependencies
-  - tqdm
-  - pytest
-  - pytest-cov
-  - coverage
   - openff-toolkit <0.11.0
-  - sphinx
-  - dicttoxml
+  - openff-units >=0.1.8
```

Comment:
I don't think we are explicitly using the openff units package. Is there a reason why we have to add it here?

Reply:
We updated `openff-units` to be backwards compatible, but the conda-forge pins sometimes don't pull in the backwards compatible version.